tests/test_shortcut.py

```diff
@@ -1,155 +1,92 @@
 import json
+from lxml import html
+
+import requests
 
 from cerberus import Validator
-from requests import request
 
 
-def test_shortcut_status_code():
-    # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    # action
-    response = request(
-        method='POST',
+def test_shortcut_status_code(purge_all_links):
+    response = requests.post(
         url='http://localhost:8888/shortcut',
         data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
     )
-    # validation
+
     assert response.status_code == 200
 
 
-def test_shortcut_body():
-    # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    # action
-    response = request(
-        method='POST',
+def test_shortcut_body(purge_all_links):
+    response = requests.post(
         url='http://localhost:8888/shortcut',
         data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
     )
-    # validation
+
     v = Validator(
         {
             'id': {
                 'type': 'string', 'allowed': [response.json()['id']]
             }
         }
     )
-    assert v.validate(response.json())
+    assert v.validate(response.json()), v.errors
 
 
-def test_shortcut_created():
-    # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    # action
-    request(
-        method='POST',
+def test_shortcut_created(purge_all_links):
+    requests.post(
         url='http://localhost:8888/shortcut',
         data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
     )
-    # validation
-    check = request(
-        method='GET',
+
+    check = requests.get(
         url='http://localhost:8888/admin/all_links'
     )
-    assert "https://github.com/Yurasb/url_shortener_testing" in json.dumps(check.json())
+    assert "https://github.com/Yurasb/url_shortener_testing" \
```

Comment:
\ - ugly. Don't use this type of line separation.

Reply:
You also could move "https://github.com/Yurasb/url_shortener_testing" to constants.